wizeline/remix-project-lab
# Feature/11/delete project

# What does this PR do?

Migrate the delete project option #11 

# How should this be manually tested?

1. Create a new project
2. Select the project
3. Click on edit
4. Click on Delete project button (You must to be admin)
5. Confirm delete



# Screenshots
<img width="1211" alt="image" src="https://user-images.githubusercontent.com/74632242/209714885-e478f1cc-a543-4722-b2ee-d3fa3615cd71.png">
<img width="600" alt="image" src="https://user-images.githubusercontent.com/74632242/209714982-a1deaf9f-f77c-4b64-91cd-3f9fefff9f3e.png">
<img width="522" alt="image" src="https://user-images.githubusercontent.com/74632242/209714997-319f1496-6f91-439d-a1d5-f8af5601ab03.png">




# Checklist
